src/app/game-state/main-loop.service.ts

```diff
@@ -103,22 +107,103 @@ export class MainLoopService {
     this.totalTicks = properties.totalTicks || 0;
     this.useBankedTicks = properties.useBankedTicks ?? true;
     this.scientificNotation = properties.scientificNotation || false;
+    this.playMusic = properties.playMusic;
   }
 
   start() {
     if (!this.characterService) {
       this.characterService = this.injector.get(CharacterService);
     }
 
-    window.setInterval(() => {
+    // The string reason we play audio is to avoid getting deprioritized in the background.
```

Comment:
I'm not sure what "the string reason" means.

Reply:
That was a typo, I've updated the comment.